### DISCUSSION TITLE
poetry update failing at updating some of the dependencies

### DISCUSSION THREAD

**Discussion by ghost** [thinkingmachines/geowrangler#148] [Enhancements Requests]
[feedback from usability tester]

I found it hard to update GW using poetry as poetry update seems to fail at updating some of the dependencies

**Comment by joshuacortez** [comment]
Closing since we're using `pip-tools` and `conda`  instead of `poetry` now. See [DEVELOPMENT.md link](https://github.com/thinkingmachines/geowrangler/blob/master/DEVELOPMENT.md)
